Current Version/final.py

```diff
@@ -0,0 +1,72 @@
+import pygame
+import pygame.surfarray as surfarray
+import numpy
+import time
+import StarField    #importing StarField effect from other module
+                    #might move all of it into final.py?
```

Comment:
You are encouraged to use imports to help structure your code, where appropriate. Pay attention to PEP conventions on module names.